db/migrate/20170509105754_create_classifications_export_segments.rb

```diff
@@ -0,0 +1,20 @@
+class CreateClassificationsExportSegments < ActiveRecord::Migration
+  def change
+    create_table :classifications_export_segments do |t|
+      t.references :project, null: false, index: true, foreign_key: true
+      t.references :workflow, null: false, index: true, foreign_key: true
+      t.integer :first_classification_id, null: false
+      t.integer :last_classification_id, null: false
+      t.integer :requester_id, null: false
+
+      t.datetime :started_at
+      t.datetime :finished_at
+
+      t.timestamps null: false
+    end
+
+    add_foreign_key :classifications_export_segments, :classifications, column: :first_classification_id
```

Comment:
Add `NOT VALID` as per https://joinhandshake.com/engineering/2017/02/27/database-migrations-and-locks.html